v63004/outref/gtm8202.txt

```diff
@@ -0,0 +1,113 @@
+#Create 3 database files for 3 regions, mumps.dat, a.dat, and b.dat for DEFAULT, AREG, and BREG respectively
+#calls gtm8202.m to set variables
+#  single region extract -SEQNO=~0,1,~2 
+#  expected to return sequences 1
+##FILTERED##%YDB-I-MUJNLSTAT, Initial processing started at ... ... .. ..:..:.. 20..
```

Comment:
Can the extract output be removed from the reference file. They make it harder to read.